eng/pipelines/mgmt-ci.yml

```diff
@@ -1,7 +1,7 @@
 trigger:
   branches:
     include:
-      - main
+      - feature/v4
```

Comment:
I suggest we just use our common pattern for both of these ci.yml files:

```
      - main
      - feature/*
      - release/*
      - hotfix/*
```

That way this pipeline will trigger for any branch follow the given pattern.

Reply:
See https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/core/ci.yml#L21-L24 as an example of the common pattern in the repo.

Reply:
@weshaggard I hope to release track1 sdk in branch `feature/v4` and the `main` branch is only for track2 sdk. So we need to disable the original pipeline for track1 in main branch.